app/app.go

```diff
@@ -450,6 +450,9 @@ func (app *App) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.Respo
 		if req.Height == app.upgradeHeightV2-1 {
 			app.SetInitialAppVersionInConsensusParams(ctx, v2)
 			app.SetAppVersion(ctx, v2)
+			if err := app.ParamsKeeper.DeleteSubspace(blobstreamtypes.ModuleName); err != nil {
+				panic(err)
+			}
```

Comment:
Note: this may be movable to a migration

Reply:
Disregard. Blobstream module can't have migrations without a refactor to how migrations are performed.

It seems a bit odd to register a migration on the params module because we're not changing the consensus version of that module.